packages/fxa-settings/src/pages/Signin/SigninTotpCode/container.tsx

```diff
@@ -41,7 +42,13 @@ export const SigninTotpCodeContainer = ({
   const signinState = getSigninState(location.state);
 
   const { queryParamModel } = useValidatedQueryParams(SigninQueryParams);
-  const { redirectTo, service } = queryParamModel;
+  const { service } = queryParamModel;
+
+  const webRedirectCheck = useWebRedirect(integration.data.redirectTo);
+
+  const redirectTo = webRedirectCheck.isValid()
+    ? integration.data.redirectTo
+    : undefined;
```

Comment:
nit: The check above uses empty string, this uses undefined. Unless there's a reason maybe we should be consistent about this.

Reply:
Sounds good! Updated to match other existing uses (`''`)